### PR TITLE
Added brief description of each column & how cards progress across th…

### PR DESCRIPTION
**Pull request by bensternthal**
…e board.

@metadave Turns out my edits/additions were very minor.

### CODE REVIEW

File: docs/how_we_work.md [hunk 1]
```diff
@@ -107,3 +107,33 @@ Open source is awesome, and sometimes we get pull requests from non-Mozillians.
...
+##### Complete
+
+The task is complete, it is _live_ and no additional work is required.
+
+
+### Workflow / How Cards Move Across The Board
+
+Cards from the queued column are pulled from left to right by whomever has taken that task. If you take a task please ensure you add yourself to the assignee list.
+
+We are currently not enforcing _Work In Progress_ limits for each column, however a good guideline is that if you should not have more than 3 cards assigned to you in progress at any given time.
```
Comment (metadave) [inline docs/how_we_work.md]: `however a good guideline is that if you should not have` => extra `if`

File: docs/how_we_work.md [hunk 2]
```diff
@@ -107,3 +107,33 @@ Open source is awesome, and sometimes we get pull requests from non-Mozillians.
 
 TODO: should we ever consider a [Developer Certificate of Origin](https://github.com/habitat-sh/habitat/blob/master/CONTRIBUTING.md#signing-your-commits). 
 
+### Github Project Columns Defined
+
+##### Backlog
+
+A list of possible tasks. Tasks in this column are not actively being worked on. In our weekly triage meeting this list is briefly discussed and if needed, prioritized. 
+
+TODO: We should have an icebox.
```
Comment (metadave) [inline docs/how_we_work.md]: suggestions on where we create this icebox? @jgmize @glogiotatidis

Comment (jgmize) [inline docs/how_we_work.md]: How about we use https://github.com/mozmar/infra/issues?utf8=%E2%9C%93&q=is%3Aopen%20is%3Aissue%20no%3Aproject for an icebox?

Comment (jgmize) [inline docs/how_we_work.md]: Or perhaps a label?

Comment (metadave) [inline docs/how_we_work.md]: `Icebox` label created.

**Pull request merged**